apps/server/internal/workspace/materializer_test.go

```
@@ -0,0 +1,312 @@
+	runGit(t, git.binary, "-C", repo, "config", "user.email", "tests@example.invalid")
+	if err := os.WriteFile(filepath.Join(repo, "README.md"), []byte("fixture\n"), 0o644); err != nil {
+		t.Fatal(err)
+	}
+	runGit(t, git.binary, "-C", repo, "add", "README.md")
+	runGit(t, git.binary, "-C", repo, "commit", "-m", "fixture")
+	return repo
+}
+
+func runGit(t *testing.T, binary string, args ...string) {
+	t.Helper()
+	cmd := exec.Command(binary, args...)
```

golangci-lint (2.13.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)